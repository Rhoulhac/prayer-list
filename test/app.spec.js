import React from 'react';
import renderer from 'react-test-renderer';
import fakeProps from '../src/fake-data';
import PrayerList from '../src/app';

describe('PrayerList', () => {
    it('should render component', () => {
        const tree = renderer.create(<PrayerList {...fakeProps} />).toJSON();

        expect(tree).toMatchSnapshot();
    });
});
