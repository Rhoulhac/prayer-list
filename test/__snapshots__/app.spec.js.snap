// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PrayerList should render component 1`] = `
<div>
  <table
    className="prayer-list-table"
  >
    <tbody>
      <tr
        className="prayer-list-header"
      >
        <th>
          First CATEGORY
        </th>
      </tr>
      <tr
        className="prayer-list-row"
      >
        <td>
          ITEM 1a
          : 
          2019-2-19
        </td>
      </tr>
      <tr
        className="prayer-list-row"
      >
        <td>
          ITEM 1b
          : 
          2019-2-18
        </td>
      </tr>
      <tr
        className="prayer-list-row"
      >
        <td>
          ITEM 1c
          : 
          2019-2-17
        </td>
      </tr>
      <tr
        className="prayer-list-header"
      >
        <th>
          Second CATEGORY
        </th>
      </tr>
      <tr
        className="prayer-list-row"
      >
        <td>
          ITEM 2a
          : 
          2019-2-18
        </td>
      </tr>
      <tr
        className="prayer-list-header"
      >
        <th>
          Last CATEGORY
        </th>
      </tr>
      <tr
        className="prayer-list-row"
      >
        <td>
          ITEM 3
          : 
          2019-2-13
        </td>
      </tr>
    </tbody>
  </table>
</div>
`;
